modules/serverless-enabling-tls-internal-traffic.adoc

```diff
@@ -41,3 +41,16 @@ spec:
 ----
 $ oc delete pod -n knative-serving --selector app=activator
 ----
+
```

Comment:
It would be great to include the example how to change the Ingress certificate. It's in the JIRA description but it's not here in this PR so it's not really clear what/how was changed.

Reply:
Thanks for the review Martin. I've included the step on changing the Ingress certificate.